Dockerfile

```diff
@@ -26,12 +26,12 @@ RUN /usr/bin/caddy -version
 RUN /usr/bin/caddy -plugins
 
 EXPOSE 80 443 2015
-VOLUME /root/.caddy /srv
+VOLUME /root/.caddy /srv /etc/caddy
 WORKDIR /srv
 
-COPY Caddyfile /etc/Caddyfile
 COPY index.html /srv/index.html
+COPY docker-entrypoint.sh /
+COPY docker-entrypoint.d /docker-entrypoint.d/
 
-ENTRYPOINT ["/usr/bin/caddy"]
-CMD ["--conf", "/etc/Caddyfile", "--log", "stdout"]
-
+ENTRYPOINT ["/bin/bash", "/docker-entrypoint.sh"]
+CMD ["/usr/bin/caddy", "--conf", "/etc/caddy/Caddyfile", "--log", "stdout"]
```

Comment:
The readme.md has a few mentions where the main config is located in the container. But this change moves it.
The documentation should be updated to remain working with this change.

How would someone use [the default config](https://github.com/abiosoft/caddy-docker#default-caddyfile) with this pull request?